src/functions/functions.ts

```diff
@@ -53,36 +43,32 @@ export const getNetworkName = (networkId: number) => {
   switch (networkId) {
     case 1:
       return 'Ethereum';
-    case 3:
-      return 'Ropsten';
-    case 4:
-      return 'Rinkeby';
     case 5:
       return 'Goerli';
```

Comment:
optional; there are a number of other Goerli networks, perhaps we could be explicit that it is Ethereum Goerli?